static/src/javascripts/projects/common/modules/identity/wizard.js

```diff
@@ -165,11 +193,11 @@ const updateSteps = (
         });
     });
 
-export const setPosition = (
+const setPosition = (
     wizardEl: HTMLElement,
-    newPosition: number,
+    unresolvedNewPosition: number | string,
```

Comment:
Under what circumstances would we expect a string here?

Reply:
you can `setPosition ` specifying a step name, i could instead do setPositionFromName if that's better!

Reply:
No, it's cool, I just couldn't see any examples anywhere of this parameter being passed as a string.

Reply:
ah yes this is for other code to pick it up! For example the implementation of this wizard on the consent journey will use it to skip between pages by name, instead of relying on numbers which is a bit flakier. 

This is the current code (hacky & messy on top of the actual issue i was testing a rough ux draft) that inspired me to do this pr, as you can see it's really coupled together with page numbers so if i were to add a splash page or something the whole thing would fall apart

    const position_consent = 0;
    const position_email = 1;

    [...]

    buttonEl.addEventListener('click',(ev:Event)=>{
        ev.preventDefault();
        const position = parseInt(wizardEl.dataset.position);
        if(position === position_email) {
            const rejectedEmailNames = getRejectedCheckboxes(checkboxesEl).map(checkboxEl => checkboxEl.getElementsByClassName('manage-account__switch-title')[0].innerText);
            if(rejectedEmailNames.length > 2 ){
                window.confirm(`You will stop receiving the following newsletters. Are you sure? : ${rejectedEmailNames.join()}`) ?
                    setPosition(wizardEl, position+1) : true;
            }
            else {
                setPosition(wizardEl, position+1);
            }
        }
        else {
            setPosition(wizardEl, position+1)
        }
    })

Reply:
Okay, so this is preliminary work to enable other changes to improve existing flaky code. You have my blessing! 😇  🙏